chart/values.yaml.tpl

```
@@ -8,6 +8,11 @@ git:
 
 hub: eu.gcr.io/freiheit-core
 tag: "$VERSION"
+log:
+  # Other possible values are "gcp" for a gcp-optimized format and "default" for json
```

```suggestion
  # Possible values are "gcp" for a gcp-optimized format and "default" for json
```
(if there are actual "other" possible values, then please mention them in the comment)